android-sdk/tasks/main.yml

```diff
@@ -64,8 +64,7 @@
   name: "Poll for Pods to become ready"
   command: oc get dc android-sdk --namespace={{ project_name }} --output jsonpath='{.status.availableReplicas}'
   register: poll_result
-  until: poll_result.stdout == "1" and poll_result.stderr == ""
-  failed_when: '"error executing jsonpath" not in poll_result.stderr and poll_result.stdout != "1" and poll_result.stdout != "0"'
```

Comment:
error executing jsonpath is needed. otherwise on OpenShift v3.3 the poller will fail because the status.availableReplicas is not present in the json until the pod comes up.